packages/cli/odahuflow/cli/parsers/local/packaging.py

```diff
@@ -137,8 +139,8 @@ def get_packager(
     packager = local.get(name)
     if not packager:
         click.echo(
-            f'The {name} packager not found in the manifest files.'
-            f' Trying to retrieve it from API server'
+            f'The {name} packager not found in the manifest files. '
```

Comment:
```suggestion
            f'The "{name}" packager is not found in the manifest files. '
```
Since It doesn't look like the short error message, I think it should have the whole sentence structure.